sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java

```diff
@@ -188,6 +192,33 @@ interface Blank extends DefinitionWithRegion<WithGroup> {
         interface WithGroup extends GroupableResource.DefinitionStages.WithGroup<WithVersion> {
         }
 
+        /**
+         * The stage of the Kubernetes cluster definition allowing to specify a tier for the managed cluster SKU.
+         */
+        interface WithPricingTier {
+
+            /**
+             * Specifies the tier of the managed cluster SKU is free.
+             *
+             * @return the next stage of the definition
+             */
+            WithCreate withFreeTier();
```

Comment:
Seems we usually name it `withFreePricingTier` or `withFreeSku`

Reply:
The name changed to `withFreeSku`,  the `withStandardTier` and `withPremiuTier` make the same fix.